src/components/NcActionCheckbox/NcActionCheckbox.vue

```diff
@@ -204,10 +204,8 @@ export default {
 		padding: 0 !important;
 		padding-right: $icon-margin !important;
 
-		// checkbox-width is 12px, border is 2
-		// (44 - 14 - 2) / 2 = 14
 		&::before {
-			margin: 0 14px 0 !important;
+			margin: 0 10px 0 8px !important;
```

Comment:
I think to make it clear that this is not a magic value (especially because this will break with Nextcloud 29 vs 30):

```suggestion
			margin-block: 0 !important;
			// 16px is the width of the checkbox including the border
			margin-inline: calc((var(--default-clickable-area) - 16px) / 2) !important;
```
---

![Bildschirmfoto_20240719_170346](https://github.com/user-attachments/assets/a4cd7b97-430e-47aa-a83b-2e311f3d6db3)

Reply:
Yeah sorry for closing the other conversation accidentally, I'm looking at this and it looks good too, so sure